bin/drop_sample_annot.py

```diff
@@ -27,12 +28,15 @@
 
 def write_sample_annotation_to_tsv(
     bam: str,
+    dna_vcf: str,
     samples: str,
+    dna_id: str,
     strandedness: str,
     single_end: str,
     sex: str,
     drop_group_sample: str,
     out_file: str,
+    gtf: str,
```

Comment:
`gtf` doesn't not seem to be used in the function below. Should it be added or can it be removed?